PlayCover/View/MenuBarView.swift

```diff
@@ -42,6 +42,12 @@ struct PlayCoverHelpMenuView: Commands {
             Button("menubar.downloadMoreApps") {
                 NSWorkspace.shared.open(URL(string: "https://ipa.playcover.workers.dev/0:/")!)
             }
+            #if DEBUG
+            Divider()
+            Button("debug.crash") {
```

Comment:
Do we really need to keep the crash button? I can't think of any reason we would need to use it.

Reply:
That's only during debug state

Reply:
True, but when will it be needed during debugging?